sample/allofhands_hawaii2016/event

```diff
@@ -0,0 +1,90 @@
+{
+	"id": 3,
+	"name": "All Hands/2016 Hawaii",
+	"latitude": null,
+	"longitude": null,
+	"location_name": "Hilton Waikoloa village, Hawaii",
+	"start_time": "2016-12-05T13:00:00",
+	"end_time": "2016-12-09T20:00:00",
+	"timezone": "Hawaii-Aleutian",
+	"description": "Multiple team meetings, happening in the same city, at the same time + some opportunity to get together as one big group as well as with other teams as it makes sense. Then, on the last day, we have a fun social event for all, Mozilla-style!",
+	"background_image": "https: //drive.google.com/uc?export=download&id=0B2b0W4GqpgLFbXRFYzdOcDVua2s",
```

Comment:
Oh, but are we looking for shorter versions of URLs? I think that isn't really a problem here. Let's wait for others to review.